js/NavMainComponent.js

```diff
@@ -415,7 +415,9 @@ NavMainComponent.prototype.moreIconClickHandler = function ($moreLink) {
     if (component.$navTertiary.find('.nav-list').hasClass('is-active')) {
         $moreLink.attr('aria-expanded', 'false');
         component.$navTertiary.removeClass('is-open');
+        component.$navQuaternary.removeClass('is-open');
         component.$navTertiary.find('.nav-list').removeClass('is-active');
+        component.$navQuaternary.find('.nav-list').removeClass('is-active');
     } else {
```

Comment:
⚠️ Micro Optimisation Alert ⚠️ – We could cache a lot of these `.find(...)` queries ahead of time. We could store a reference to these nodes on load which would save performing them each time the event fires. However this comes with the caveat that the menu markup would not change during it's lifespan (which I think probably _is_ the case)

I totally understand if we do not want to change it to implement this ☝️

Reply:
`53 occurrences`

![](https://i.kym-cdn.com/photos/images/original/000/284/529/e65.gif)

Reply:
53, yikes. Maybe a candidate for some love if we ever get into any FE performance issues.